server/store/datastore/migration/001_add_org_id.go

```diff
@@ -26,12 +26,12 @@ import (
 var addOrgID = xormigrate.Migration{
 	ID: "add-org-id",
 	MigrateSession: func(sess *xorm.Session) error {
-		if err := sess.Sync(new(userV031)); err != nil {
+		if err := sess.Sync(new(userV009)); err != nil {
```

Comment:
1. Why `userV009`, not `userV001`?
2. Shouldn't the old records from 'migration' table be dropped also?
3. Maybe just drop old migration code without renaming?

Reply:
1. I thought it should be the same as filename: `019_drop_old_cols.go` - `oldPipeline019`, `022_add_orgs.go` - `repo022`. But it doesn't matter a lot :)